storage/remote/queue_manager.go

```diff
@@ -1325,6 +1380,7 @@ func (s *shards) runShard(ctx context.Context, shardID int, queue *queue) {
 			droppedSamples := int(s.enqueuedSamples.Load())
 			droppedExemplars := int(s.enqueuedExemplars.Load())
 			droppedHistograms := int(s.enqueuedHistograms.Load())
+			droppedFloatHistograms := int(s.enqueuedFloatHistograms.Load())
 			s.qm.metrics.pendingSamples.Sub(float64(droppedSamples))
 			s.qm.metrics.pendingExemplars.Sub(float64(droppedExemplars))
 			s.qm.metrics.pendingHistograms.Sub(float64(droppedHistograms))
```

Comment:
This should be done for float histograms too Same with `failedHistogramsTotal` below and these places:

* https://github.com/prometheus/prometheus/blob/87368d69d82be6846ebefe36b985a4ec542fec88/storage/remote/queue_manager.go#L1529

* https://github.com/prometheus/prometheus/blob/87368d69d82be6846ebefe36b985a4ec542fec88/storage/remote/queue_manager.go#L1544